lib/stripe/util.rb

```diff
@@ -54,6 +54,7 @@ def self.object_classes
         'order' => Order,
         'order_return' => OrderReturn,
         'three_d_secure' => ThreeDSecure,
+        'apple_pay_domain' => ApplePay::Domain,
```

Comment:
+1 `ApplePayDomain`. I don't think this is going to get too unwieldy within the top-level namespace anytime soon.